setup.py

```diff
@@ -30,7 +30,7 @@
     packages=find_packages(exclude=['tests']),
```

Comment:
We should change the `classifiers` parameter above in this very same file to replace

```
        'Programming Language :: Python :: 2',
        'Programming Language :: Python :: 2.6',
        'Programming Language :: Python :: 2.7',
```

by 

``` 
        'Programming Language :: Python :: 3',
```